code/game/objects/structures/flora.dm

```diff
@@ -77,6 +84,7 @@
 //bushes
 /obj/structure/flora/bush
 	name = "bush"
+	desc = "A medium-sized plant. It may have products growing on it such as fruits, or it may just be empty."
```

Comment:
Awkward sentence

Reply:
I was indeed a bit in a confusion as to what to put there. Open for suggestions.